DDG4/src/Geant4InputAction.cpp

```diff
@@ -29,14 +29,28 @@ typedef Geant4InputAction::Vertices Vertices ;
 
 /// Initializing constructor
 Geant4EventReader::Geant4EventReader(const std::string& nam)
-  : m_name(nam), m_directAccess(false), m_currEvent(0)
+  : m_name(nam), m_directAccess(false), m_currEvent(0), m_inputAction(0)
 {
 }
 
 /// Default destructor
 Geant4EventReader::~Geant4EventReader()   {
 }
 
+/// Get the context (from the input action)
+Geant4Context* Geant4EventReader::context() const {
+  if( 0 == m_inputAction ) {
+    printout(FATAL,"Geant4EventReader: %s", "No input action registered!");
```

Comment:
the "%s" has to be removed, second argument is the "name for the printout
From the failed test (https://travis-ci.org/AIDASoft/DD4hep/jobs/346807978#L1623)
```
Geant4EventReader: ��҂ FATAL No input action registered!
```